### PR TITLE
the-next-web-clone-feature-branch

### PR DESCRIPTION
**Pull request by rayking12**
cloned the next web with responsive web design.
Tasks done:
created HTML file.
created CSS file
Added linters.
Added styles to the HTML texts and img.
positioned img and texts.
added grid design to the img and texts.
Added media queries to the designs for tablets and Phones.
Added footer.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1000 @@
...
+  font-family: Georgia, "Times New Roman", Times, serif;
+  display: -ms-grid;
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style-type: none !important;
```
Comment (MuriloRoque) [inline style.css]: - [ ] Please remove this `!important`, it's not a good practice.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @rayking12 and @Byabasaija :wave: 

### Your page is almost like the original, you have done a great job in this one :clap: 

### Please follow my improvement suggestions below:

## Large screen

- [x] Your second navbar needs to be smaller, take a look at the original:

**Original:**

Screenshot from 2020-10-29 15-21-21

**Yours:**

Screenshot from 2020-10-29 15-21-18

- [ ] Your navbar should be fixed when we scroll down:

Screenshot from 2020-10-29 15-21-02

- [ ] Please adjust your second image in these articles, it looks higher than the others:

Screenshot from 2020-10-29 15-20-11

- [x] Please work a little more on your footer styling, it doesn't look like the original. Pay 
special attention to the colors:

**Original:**

Screenshot from 2020-10-29 15-20-44

**Yours:**

Screenshot from 2020-10-29 15-20-43

## Medium screen

- [ ] The links below the 'Latest Funding' section title need to be below it, not to the side:

**Original:**

Screenshot from 2020-10-29 15-21-41

**Yours:**

Screenshot from 2020-10-29 15-21-44

## Small screen

- [ ] Your top articles should have the same height, just like the one in the middle, please fix the first and the last one:

Screenshot from 2020-10-29 15-22-25

- [ ] In your 'Latest News' section, the images should have a bigger height, and the text should be aligned according to the original, check below:

**Original:**

Screenshot from 2020-10-29 15-22-53

**Yours:**

Screenshot from 2020-10-29 15-22-50

- [ ] In your 'Latest Deals' section, the images should have a bigger height, and the text should be aligned according to the original, check below:

**Original:**

Screenshot from 2020-10-29 15-23-40


**Yours:**

Screenshot from 2020-10-29 15-23-42

## General

- [ ] Please check my comment in your code below.

- [ ] In your CSS, I detected a bad practice of repeating a lot of the same property. I suggest that you create specific classes for those rules and then apply the classes to the elements you want to have them. For example, create a d-flex class with display: flex, then give the elements you want 'flexed' in this class. Check below the number of times you are repeating those rules:

      20 |   display: block;
      18 |   background-position: 50%;
      18 |   width: 100%;
      17 |   display: flex;
      16 |   height: 50%;
      15 |   background-size: cover;
      10 |   display: grid;
        9 |   font-weight: 700;

## Summary
- Your page is great, it's almost like the original :+1: 
- Your repository looks professional :muscle: 
- We just need to fix some details before moving forward :rocket: 

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

File: style.css (hunk 1 above)
Comment (byabasaija) [reply style.css]: Thank you for your feedback, we will improve it as soon as possible.

**Review by uryelah — APPROVED** [review]
Hi **Pascal Byabasaija & King Ray**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job following the previous TSE recommendations!

Your project looks great, I especially liked how you worked on your footer and made sure to include even the hover effects in the icons!

You have the minimum requirements to go to the next project, but before that make sure to check the optional suggestions left below. You can work on them in your spare time or just look up the suggestions for learning's sake.

Good luck with your next HTML&CSS projects! 🚀  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---------

### Optional suggestions

- [ ] [OPTIONAL] The navbar is still not fixed at the top of the page. 
It seems like you have a couple of issues when trying to make it work.
For this specific navbar to show up fixed in the top the properties you'll need to set are:
  - `position` to fixed
  - `z-index` to a high number such as 100 so it shows up above the articles. [The `z-index` property sets the layer the element is rendered at](https://css-tricks.com/almanac/properties/z/z-index/), the bigger the number the most it will be at the top of other elements. (Whenever you work on a fixed or absolute element and it 'disappears' check if it's not a z-index issue.)
  - `background-color` to white, since it's transparent at the moment
  - `width` to 100vw of the window. The `vw` unit means basically 100% of the viewport width, it also works for the window height with `vh`
  - And last but not least remember to give the main content after the nav some margin or padding top to take into account the new positioning of the navbar.


- [ ] [OPTIONAL] The Images at the latest part of the Topics Sections(Hard Fork, Apps, Tech, etc) are distorted. That's happening because you are asigning them fixed `width` and `height` properties and they are a direct child of a flex container.
You have two ways to fix this:
  - Remove the image tag and instead create a div tag that has the width and height you want, a `background-image` property containing the image link and then set the `background-size`, `background-position`, and `background-repeat` properties of the div in such a way the image `covers` the size of the div and is positioned at the center of it and does ont repeat itself. [(More on CSS background properties here)](https://www.w3schools.com/cssref/css3_pr_background.asp)
  - Wrap the image tag with a div around with that has the width and height you want plus an `overflow: hidden-property` and then set either the width or the height of the image in a way it takes all the content space of the div.

---------

    
    
**Keep being an awesome dev and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**